pkgs/by-name/ko/koboldcpp/package.nix

```diff
@@ -114,6 +120,7 @@ effectiveStdenv.mkDerivation (finalAttrs: {
     (makeBool "LLAMA_CLBLAST" clblastSupport)
     (makeBool "LLAMA_VULKAN" vulkanSupport)
     (makeBool "LLAMA_METAL" metalSupport)
+    (lib.optionalString (nvidiaArch != "") "CUDA_DOCKER_ARCH=${nvidiaArch}")
```

Comment:
Code-searching the upstream repo, I don't see any references to the variable other than https://github.com/LostRuins/koboldcpp/blob/7499a6bd4b714aa1bb43284ed87ccf072faa3825/Makefile#L172-L174

Makes me wonder if we should instead set NVCC_APPEND_FLAGS so we can represent a _list_ of gencodes (`cudaPackages.flags.gencodes`).

Reply:
```Make
ifdef CUDA_DOCKER_ARCH
	NVCCFLAGS += -Wno-deprecated-gpu-targets -arch=$(CUDA_DOCKER_ARCH)
else
```

As far as I can tell setting `CUDA_DOCKER_ARCH` will set `NVCCFLAGS` for us, isn't that correct?

Reply:
> cudaPackages.flags.gencodes

I search on nixpkgs (https://github.com/search?q=repo%3ANixOS%2Fnixpkgs%20cudaPackages.flags.gencodes&type=code) can couldn't find any reference to `cudaPackages.flags.gencodes` anywhere (Besides this PR)?